frontend/src/lib/MetricsManager.ts

```diff
@@ -18,7 +18,7 @@
 import { IS_DEV_ENV, IS_SHARED_REPORT } from "./baseconsts"
 import { SessionInfo } from "lib/SessionInfo"
 import { logAlways } from "./log"
-
+import { segmentLauncher } from "./Segment"
 /**
```

Comment:
Minor: If we intend to load Segment from Segment.ts instead of index.html, let's update the comment here to mention this.

(Also: let's leave the space above between the imports and the first comment)

Reply:
Updated the comment to reflect changes. 
Yes, I did not wish to disturb the segment io flow since it has been working fine till now. But, we now have better control over their api methods.